tests/scipy_stats_test.py

```diff
@@ -1435,6 +1435,249 @@ def testMultivariateNormalLogpdfBatch(self, ndim, nbatch, dtype):
     result2 = jax.vmap(lsp_stats.multivariate_normal.logpdf)(x, mean, cov)
     self.assertArraysAllClose(result1, result2, check_dtypes=False)
 
+
+  # gumbel dist tests, for all these tol is set to 5e-3 because scipy uses 64-bit float
```

Comment:
scipy uses 64-bit for all distributions, so I'm not sure why there is need to mention it here.